index.js

```diff
@@ -9,14 +9,23 @@ function search(dir) {
   var context = path.resolve(dir)
   if (!fs.existsSync(context)) return []
   var contents = fs.readdirSync(context)
-  return contents.map(function(name) {
+  var scopedRoots = []
```

Comment:
@ryanve good catch, the scope should definitely be in the output, I'll get that fixed.

If you feel strong about the indicator I can add it but think it would cut some usability if you wanted to pipe this command to another command (`rimraf` for example). Think it would make for nice content to a `--pretty` or `--verbose` flag at some point in future. Conversely, a flag could be passed down the line to specify only raw output. Whichever road you prefer is cool.

Reply:
@ryanve just realized what you meant by `found.scope =`, going to add that. Can ignore the comment above.